static/usr/lib/systemd/system/plymouth-start.service.d/core-override.conf

```diff
@@ -0,0 +1,9 @@
+[Unit]
+After=snap-initramfs-mounts.service
```

Comment:
This unit does not exist in core-base, and if did exist would create the same cycle as in core-initrd.

Reply:
Yeah, good point. I've removed it now.